### ISSUE TITLE
[FR]: TT-Rss - Import starred articles

### ISSUE THREAD

**Issue by dariottolo** [martinrotter/rssguard#963]
### Brief description of the feature request

I am using RSS Guard to connect to a selfhosted istance of tt-rss.
It works beautifully, but I noticed that starring articles works only one way. If I mark an articole important in RSS Guard, I can see it starred in tt rss.
Unfortunately RSS Guard does not seem to retrieve starred articles from tt rss.
I believe it would be possible through the apis (https://tt-rss.org/wiki/ApiReference --> getFeeds --> -1 Special (e.g. Starred, Published, Archived, etc.)
Is it something that you can implement?
Thank you very much in advance for your attention.
Regards

**Comment by martinrotter** [comment]
Let me test.

**Comment by martinrotter** [comment]
Double checked and "starred" state is synchronized both ways. Make a screenshot of your TT-RSS "edit" account dialog from RSS Guard to see what options you have activated.

**Comment by dariottolo** [comment]
Thanks for your reply. Below the screenshot requested:
![edit](https://github.com/martinrotter/rssguard/assets/23664193/a6b053fd-87ad-4675-a924-22fbda227e24)
I activated the "intelligent synchronisation algorithm", but I see no difference in regards to starred articles.
Below the articles I currently see in the important category:
![important](https://github.com/martinrotter/rssguard/assets/23664193/ed1019fc-29a0-4d90-94b9-ec21aeb30502)
Those were starred by me in RSS Guard today. If I restart RSS Guard, I see them no more.
And last screenshot is TT-RSS, where I now have 451 unred and starred articles:
![tt-rss](https://github.com/martinrotter/rssguard/assets/23664193/e511aea9-e8ea-49ac-a14b-8c5c026e16c4)

**Comment by dariottolo** [comment]
Done. Details have been sent to your email

**Comment by martinrotter** [comment]
Closing, if issue persists, re-send me the credentials please, I will test.

I do not want to install TT-RSS via docker so lately I have limited options for TT-RSS testing.

**Comment by dariottolo** [comment]
I am using:
<h4>RSS Guard</h4><b>Version:</b> 4.8.6 (built on Windows/AMD64)<br/><b>Revision:</b> ff032516b<br/><b>Build date:</b> 07/09/2025 09:59<br/><b>OS:</b> Windows 11 Version 25H2<br/><b>Qt:</b> 6.8.3 (compiled against 6.8.3)
it does not seem to us that the issue is resolved.
I can provide you with a login to tt-rss.
Let me know if you are still interested in looking into this.

**Comment by martinrotter** [comment]
Yes, i would very much like login to some tt-rss instance for testing purposes as I do not have any other way to test tt-rss right now.
